client_cli/src/main.rs

```diff
@@ -159,21 +161,20 @@ fn main() -> Result<()> {
 /// Fails if submitting over network fails
 #[allow(clippy::shadow_unrelated)]
 pub fn submit(
-    instruction: impl Into<Instruction>,
+    instructions: Executable,
```

Comment:
What about `impl Into<Executable>`? This would allow to submit on `Vec<Transaction>` directly.

Reply:
Very helpful thanks